content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -50,8 +50,8 @@ Creating a NetworkPolicy resource without a controller that implements it will h
 
 There are two sorts of isolation for a pod: isolation for egress, and isolation for ingress.
 They concern what connections may be established. "Isolation" here is not absolute, rather it
-means "some restrictions apply". The alternative, "non-isolated for $direction", means that no
-restrictions apply in the stated direction.  The two sorts of isolation (or not) are declared
+means "some restrictions apply". The alternative, "non-isolated for direction", means that no
```

Comment:
Hello @fracappa. A couple of comments:
There are many, minor changes in this pull request that were added possibly by your editor. If the style or formatting of the page is correct, we recommend that you do **not** change the content.

In this paragraph, I agree that the `$` (dollar sign) is not entirely clear.

Perhaps the sentence could be reworded:
The alternative is "non-isolated for `<direction>`" where direction is either egress or ingress. No restrictions apply in the stated direction.